src/main/java/com/minecolonies/achievements/MineColoniesAchievement.java

```diff
@@ -40,8 +38,7 @@ public MineColoniesAchievement(final String id, final String name, final int off
      * @param blockIcon The icon to show
      * @param parent    the parent achievement
      */
-    public MineColoniesAchievement(final String id, final String name, final int offsetX, final int offsetY, final Block blockIcon, final Achievement parent)
-    {
+    public MineColoniesAchievement(final String id, final String name, final int offsetX, final int offsetY, final Block blockIcon, final Achievement parent) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)